deployment/hasura/migrations/AerieMerlin/16_associate_external_datasets_with_simulation/up.sql

```diff
@@ -0,0 +1,16 @@
+alter table plan_dataset
+  add column associated_simulation_dataset_id integer default null,
```

Comment:
Minor, but you don't need to specify `default null` 🙂 
```suggestion
  add column associated_simulation_dataset_id integer,
```

Reply:
change has been made